config/config.json

```diff
@@ -36,7 +36,7 @@
           {"cert-fio38-c" : true },
           {"cert-dcl58-cpp" : true },
           {"google-global-names-in-headers" : true },
-          {"google-runtime-memset" : true },
+          {"performance-inefficient-vector-operation" : true },
```

Comment:
Let's keep the alphabetical ordering here...

Reply:
Oh, I also turned on the wrong checker here.